src/components/Header/Header.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { ReactComponent as Logo } from "../../assets/home.svg";
+import styled, { ThemeProvider } from "styled-components";
+import { AppRoutes } from "../../configs/AppRoutes";
+import { CustomColors } from "../../colors/Colors";
+
+const StyledMenu = styled.div`
+  width: 85%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+
+  &:hover  {
+    cursor: pointer;
+  }
+`;
+
+const StyledOption = styled(Link)`
```

Comment:
já subjektivně a osobně preferuji psát názvy styled componet jako 
```
{KOMPONENA}_{SUFIX}
```
například:
```
Link_Option = 
```

pak je hned jasný, ž4e je to styled componenta a je jasný z čeho dědí vlasnosti (z linku)

Reply:
Ok, beru v potaz. Urobím si research vrámci rôznych repozitárov a prípadne zmením pozdejc ;)